src/BoilerPlate/Game.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "Player.hpp"
+#include "Asteroid.hpp"
+class Game
+{
+public:
+
+	//Constructor
+	Game();
+	~Game();
+
+	//Functions
+	void RenderGame();
+	void UpdateGame(double, double, double);
+	void DebuggingLine();
+	void PushAsteroids();
+
+	//Members
+	Player player_;
+	vector<Asteroid> asteroids_;
```

Comment:
it seems like you included an using statement. Please do not do this, we already discussed it in class.